app/main.py

```diff
@@ -1,13 +1,13 @@
-# from models import Actor
-# from managers import ActorManager
-#
-# if __name__ == "__main__":
-#     Actor.objects = ActorManager()
-#
-#     Actor.objects.create(first_name="Emma", last_name="Watson")
-#     Actor.objects.create(first_name="Daniel", last_name="Radclife")
-#     print(Actor.objects.all())
-#     Actor.objects.update(2, "Daniel", "Radcliffe")
-#     print(Actor.objects.all())
-#     Actor.objects.delete(1)
-#     print(Actor.objects.all())
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
```

Comment:
Ensure that the `ActorManager` class is using the correct table name `actors` as per the task requirements. Currently, it is set to `cinemas`, which is incorrect. Update the `ActorManager` class to use the correct table name to avoid any database-related issues.